src/vs/workbench/browser/actions/helpActions.ts

```diff
@@ -242,7 +242,10 @@ class OpenRequestFeatureUrlAction extends Action2 {
 class OpenLicenseUrlAction extends Action2 {
 
 	static readonly ID = 'workbench.action.openLicenseUrl';
-	static readonly AVAILABLE = !!(isWeb ? product.serverLicense : product.licenseUrl);
+	// --- Start Positron ---
+	// modify to be available in web by checking product.serverLicenseUrl instead of product.serverLicense
+	static readonly AVAILABLE = !!(isWeb ? product.serverLicenseUrl : product.licenseUrl);
```

Comment:
are we able to check `isWeb && !!process.env.RS_SERVER_URL` so that the server license is presented specifically in workbench?

Reply:
Yes, that is possible. I recall a discussion somewhere that it would be ok for now to show this for all web deployments. I think we might need to do something extra for the cases where it's web and not Workbench.

Maybe a context key should be set or `serverLicenseUrl` is only added to the `product.json` at build time. I would want something that is set similarly across the different types of web uses.